test/rest.middleware.test.js

```diff
@@ -99,6 +99,15 @@ describe('loopback.rest', function() {
       .expect(200, done);
   });
 
+  it("allows models to provide a custom HTTP path", function(done) {
+    var ds = app.dataSource('db', { connector: loopback.Memory });
+    var CustomModel = ds.createModel('CustomModel', {name: String}, {"http": { "path": "domain1/CustomModelPath" } });
```

Comment:
We try to keep the lines at most 80 characters long, that way the diffs can be reviewed without horizontal scrolling.

Please split this line.